dso_pkg/dso/policy/rnn_policy.py

```
@@ -142,6 +142,42 @@ def safe_cross_entropy(p, logq, axis=-1):
+            dummy_state = self.controller.get_initial_state(1)
+            if isinstance(dummy_state, list):
+                self._state_size = [s.shape[-1] for s in dummy_state]
+            else:
+                self._state_size = dummy_state.shape[-1]
+        return self._state_size
+    
+    @property
+    def output_size(self):
+        return self._output_size
+    
+    def get_initial_state(self, batch_size, dtype=tf.float32):
```

Ruff (0.12.2)

Unused method argument: `dtype`

(ARG002)